radis/lbl/broadening.py

```diff
@@ -2178,23 +2197,8 @@ def _broaden_lines(self, df):
                             estimated_time
                         )
                     )
-                if chunksize is None:
-
-                    # Deal with all lines directly (usually faster)
```

Comment:
did you remove this comment from the code? if so, add it back

Reply:
Sorry, I might have removed it by mistake. Adding it back.